exporters/elasticsearch-exporter/src/main/resources/zeebe-record-template.json

```diff
@@ -47,6 +47,9 @@
         },
         "sequence": {
           "type": "long"
+        },
+        "tenantId": {
+          "type": "keyword"
```

Comment:
:x: Should we include the `tenantId` property in the general `zeebe-record` template?

From my understanding, this template is extended by other templates, and it includes the metadata properties. Wouldn't  we duplicate the `tenantId` property in each template if we place it here as well?

Reply:
Oops, this is simply a mistake on my part. I'll remove it